string robot_id
string path